source/2019-06-28-the-ember-times-issue-104.md

```diff
@@ -98,17 +99,19 @@ Halò Emberistas! 🐹
 <p>This week we'd like to thank our siblings for their contributions to Ember and related repositories! 💖</p>
 
 ---
-
-## [Got a Question? Ask Readers' Questions! 🤓](https://docs.google.com/forms/d/e/1FAIpQLScqu7Lw_9cIkRtAiXKitgkAo4xX_pV1pdCfMJgIr6Py1V-9Og/viewform)
+## [Readers' Questions: What is Embroider?](https://discuss.emberjs.com/t/readers-questions-what-is-embroider/16722)
```

Comment:
I think you can display this section at the very top, in case there are readers who don't know Embroider well. Placing it on top would help them understand what Embroider is, before we ask them to read further with other Embroider-related articles. What do you think?

Reply:
Yes that sounds good! I'll edit that once I'm back at a computer tomorrow morning

Reply:
That sounds great...I'm going to try moving this myself, so can get a head start on the issue

Reply:
Thank so much @amyrlam :sparkles: